models/sale_artwork.coffee

```diff
@@ -58,6 +58,9 @@ module.exports = class SaleArtwork extends Backbone.Model
     count += if n is 1 then '' else 's'
     count
 
+  formatBidCount: ->
+    if @bidCount() is '0 bids' then '' else "(#{@bidCount()})"
```

Comment:
I'd suggest checking the number of bids itself, rather than the string representation.

Are we using `bidCount` directly anymore? If not, maybe we should just redefine it, rather than introducing a new method.